tests/garage/torch/algos/test_loss_function_utils.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.algos.loss_function_utils as torch_loss_utils
+from tests.fixtures import TfGraphTestCase
+
+
+def stack(arr, d):
+    return np.repeat(np.expand_dims(arr, axis=0), repeats=d, axis=0)
+
+
+arrange = np.arange(6)
+pi_digits = [3, 1, 4, 1, 5, 9]
+e_digits = [2, 7, 1, 8, 2, 8]
+fibs = [1, 1, 2, 3, 5, 8]
+
+
+class TestLossFunctionUtils(TfGraphTestCase):
+    # yapf: disable
+    @pytest.mark.parametrize('gae_lambda, rewards_val, baselines_val', [
+        (0.4, np.ones((4, 6)), np.zeros((4, 6))),
+        (0.8, stack(pi_digits, 4), stack(arrange, 4)),
```

Comment:
is this really simpler than just defining them as static arrays at the top of the file?

Reply:
fixed